handbook/company/pricing-features-table.yml

```diff
@@ -371,12 +371,15 @@
 - categoryName: Device management
   features:
     - industryName: Interactive MDM migration #   « end-user initiated MDM migration, with interactive UI
-      tier: Free
+      tier: Premium
```

Comment:
MIKE: Isn't end user initiated migration free?

Reply:
@noahtalerman response from MIke👆

Reply:
@mikermcneil @Sampfluger88 it's paid only because it only works for DEP Macs (DEP is paid).

We can make it free later. We just prioritized DEP only to move quickly for our gaming company customer.